ppr-ui/src/components/tombstone/TombstoneDefault.vue

```diff
@@ -3,9 +3,28 @@
     <v-col v-if="isStaff" class="staff-header" cols="1"></v-col>
     <v-col :cols="isStaff? '11' : '12'" :class="isStaff? 'pl-4' : ''">
       <div class="ma-0 pa-0">
-        <span class="tombstone-header">
-          <b>{{ header }}</b>
-        </span>
+        <v-row no-gutters class="justify-space-between align-baseline">
+          <span class="tombstone-header">
+            <b>{{ header }}</b>
+          </span>
+          <v-tooltip top content-class="top-tooltip pa-5" nudge-left="30">
+            <template  v-slot:activator="{ on, attrs }">
+              <a :href="'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/'
+              +'permits-licences/news-updates/modernization-updates/modernization-resources#ppr'"
```

Comment:
Minor indent here.
Linter probably isn't picking it up, but it helps with readability.